doc/dialects/hive.md

```diff
@@ -280,6 +280,43 @@ at java.base/java.lang.reflect.Method.invoke(Method.java:566)
 ... 23 more
 ```
 
+### Kerberos alias usage instead of principal
+
+In the Java JDK version `1.8.0_242`, we found out that the Kerberos client alias is used instead of the Kerberos principal. This can cause the permission issues since alias does not have access privileges, for example, when connecting to the Hive server.
```

Comment:
```suggestion
In the Java JDK version `1.8.0_242`, we found out that the Kerberos client alias is used instead of the Kerberos principal. This can cause permission issues since alias does not have access privileges, for example, when connecting to the Hive server.
```